sonar-plugin-api/src/main/java/org/sonar/api/CoreProperties.java

```diff
@@ -499,7 +499,9 @@
 
   /**
    * @since 4.5
+   * @deprecated no used anymore since 5.2
    */
+  @Deprecated
   String SIZE_METRIC = "sonar.technicalDebt.sizeMetric";
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Do not forget to remove this deprecated code someday. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1133)